samples/sample-04/src/main/resources/application.yml

```diff
@@ -0,0 +1,24 @@
+spring.datasource.url: jdbc:mysql://localhost:3306/notes
+spring.datasource.username: root
+spring.datasource.password:
+
+logging.level.org.hibernate.SQL: debug
+
+spring.jpa.database:  MYSQL
+spring.jpa.open-in-view: true
+spring.jpa.show-sql:  true
+spring.data.jpa.repositories.bootstrap-mode: default
+
+spring.jpa.database-platform: org.hibernate.dialect.MySQL5Dialect
+spring.jpa.hibernate.ddl-auto: update
+
+logging.level.org.springframework: DEBUG
+spring:
+  datasource:
+    driver-class-name: com.mysql.cj.jdbc.Driver
+
+spring.kafka.bootstrap-servers: localhost:9092
+spring.kafka.consumer.group-id: noteGroup
+spring.kafka.consumer.auto-offset-reset: earliest
+spring.kafka.consumer.properties.spring.json.trusted.packages: com.remusrd.notesample.domain.event
```

Comment:
Can you reformat this file to really true YAML style?

Reply:
Thanks a lot for the feedback, will fix it in the next days!